### PR TITLE
[Network]hide create for management-ip-config create

### LINKED ISSUE #1270: Firewall management IP configuration should only be created while creating a new firewall
- If the issue is to do with Azure CLI 2.0 in-particular, create an issue here at [Azure/azure-cli](https://github.com/Azure/azure-cli/issues)

### Extension name (the extension in question)
Network (firewall)

### Description of issue (in as much detail as possible)
Currently the only way to attach a Management IP Configuration appears to be with this workflow:
az network firewall create 
az network firewall ip-config create
az network firewall management-ip-config create

Each of these commands executes a PUT against firewall. 
However, this cannot work, as creating a firewall with forced tunneling requires both the “data” and “management” IP configs to be attached to the firewall at the same time.

Attaching the data ip-config first, and then management ip-config later becomes a migration scenario ion the back end, which is not supported today.

Further, we need to prevent customers from attempting to modify already existing firewalls and attaching management IP configs to them, for the same reason.
(There is work planned to block this scenario in NRP, but it’s not there yet.)

Basically, attaching a management IP configuration should only be possible one time, at firewall creation time, and it requires both types of ip-config in one PUT call.

Customers should be able to add more “data” IP configs. (but only a single “management”  IP config)
Additionally, customers should be able to change the public IP referenced in the “management” IP config. 

-----

### PR DESCRIPTION
**Pull request by MyronFanQiu**
Due to service limitation. hide management-ip-config create.

1. should we hide this command or not?  We can just left here and told user that it cannot work now.

Fix #1270 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S165.

File: src/azure-firewall/azext_firewall/tests/latest/test_azure_firewall_scenario.py [hunk 1]
```diff
@@ -95,22 +95,27 @@ def test_azure_firewall_management_ip_config(self, resource_group):
...
 
+        '''
+        # Disable it due to service limitation.
         self.cmd(
             'network firewall management-ip-config create -g {rg} -n {management_ipconfig} -f {af} --public-ip-address {management_pubip} --vnet-name {management_vnet}',
             checks=[
                 self.check('name', '{management_ipconfig}'),
                 self.check('subnet.id', subnet_id_management_ip_config)
             ])
+        '''
```
Comment (haroldrandom) [inline src/azure-firewall/azext_firewall/tests/latest/test_azure_firewall_scenario.py]: I think it's not a convention to comment via DocString style. how about replace it with `#`?

Comment (MyronFanQiu) [reply src/azure-firewall/azext_firewall/tests/latest/test_azure_firewall_scenario.py]: fix

File: src/azure-firewall/azext_firewall/_params.py [hunk 2]
```diff
@@ -113,6 +113,18 @@ def load_arguments(self, _):
...
+        c.argument('management_item_name', options_list=['--m-name'], arg_group="Management Ip Config",
+                   help='Name of the management IP configuration.', is_preview=True)
+        c.argument('management_subnet', validator=get_management_subnet_validator(), help=argparse.SUPPRESS, is_preview=True)
+        c.argument('management_virtual_network_name', virtual_network_name_type, options_list=['--m-vnet-name'],
+                   arg_group="Management Ip Config",
+                   help='The virtual network (VNet) name for management ip configuation. '
+                        'It should contain one subnet called "AzureFirewallManagementSubnet".', is_preview=True)
+        c.argument('management_public_ip_address', help='Name or ID of the public IP to use for management ip configuation.',
+                   arg_group="Management Ip Config", options_list=['--m-public-ip-address'],
+                   validator=get_management_public_ip_validator(), is_preview=True)
```
Comment (haroldrandom) [inline src/azure-firewall/azext_firewall/_params.py]: need `--m` prefix? Can we removed it? I don't know whether it will cause coflict?

Comment (MyronFanQiu) [reply src/azure-firewall/azext_firewall/_params.py]: no. We already have --public-ip-address in ip-config create

Comment (MyronFanQiu) [reply src/azure-firewall/azext_firewall/_params.py]: we have four almost same arguments. The only difference is the `m-` prefix.

Comment (haroldrandom) [reply src/azure-firewall/azext_firewall/_params.py]: How about group them in a argument group **Management** to give user an instinct hint?

Comment (MyronFanQiu) [reply src/azure-firewall/azext_firewall/_params.py]: Have done

**Pull request merged**